Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -392,6 +404,23 @@ void CookieStore::set(CookieInit&& options, Ref<DeferredPromise>&& promise)
     Cookie cookie;
     cookie.name = WTFMove(options.name);
     cookie.value = WTFMove(options.value);
+
+    if (!cookie.name.length()) {
```

Comment:
I think you can use `isEmpty` here and below

Reply:
Fixed1